dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -162,6 +165,10 @@ object Datadog {
         initialized = false
     }
 
+    fun setVerbosity(level: Int) {
```

Comment:
this needs to be documented, especially the values that the level can take. Is't there an IntDef related annotation that we can use in Kotlin ?

Reply:
Yes indeed

Reply:
Done